### PR TITLE
Docs/reorganize getting started

### PR DESCRIPTION
**Pull request by jBorkowska**

### CODE REVIEW

File: docs/index.mdx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Main
+title: Patrol
```
Comment (bartekpacia) [inline docs/index.mdx:2]: These titles are also used for analytics. If we change them, we'll break it (at least for some time)

Comment (jBorkowska) [reply docs/index.mdx:2]: I think it's better to fix it earlier then.

Comment (jBorkowska) [reply docs/index.mdx:2]: You can propose another title for this page as well, I'm not happy with the new one actually.

Comment (bartekpacia) [reply docs/index.mdx:2]: > I think it's better to fix it earlier then.

Yeah, I agree. Let's just make sure that business is okay with it.

> You can propose another title for this page as well, I'm not happy with the new one actually.

TBH, I have no idea as well 😅 I don't like both names.

File: docs/getting-started/getting-started.mdx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Patrol consists of 2 packages:
+
+- [patrol] – the core package
+- [patrol_cli] - the CLI app which enables the [native automation feature]
+
+There are two ways of using Patrol - to create **E2E UI tests** which contain 
+testing **native features of your app** (eg. localization permissions) or to **enhance** your experience of writing 
+**widget tests** by making use of our **custom finders**. Depending of which way you want to go, 
+head over to [native setup] or [custom finders setup].
```
Comment (bartekpacia) [inline docs/getting-started/getting-started.mdx:15]: This paragraph sounds like these choices are mutually exclusive (when they're complementary).

File: docs.json [hunk 3]
```diff
@@ -14,7 +14,14 @@
   },
   "sidebar": [
     ["About", "/"],
-    ["Getting started", "/getting-started"],
+    [
+      "Getting started",
+      [
+        ["Getting started", "/getting-started/getting-started"],
+        ["Native setup", "/getting-started/native"],
```
Comment (Albert221) [inline docs.json:21]:
```suggestion
        ["Native automation", "/getting-started/native"],
```

File: docs.json [hunk 4]
```diff
@@ -14,7 +14,14 @@
...
+        ["Custom finders setup", "/getting-started/custom-finders"]
```
Comment (Albert221) [inline docs.json:22]:
```suggestion
        ["Custom finders", "/getting-started/custom-finders"]
```

File: docs/index.mdx (hunk 1 above)
Comment (Albert221) [reply docs/index.mdx:2]: `Patrol - Flutter testing framework`? It's the title of the home page - https://patrol.leancode.co

Comment (bartekpacia) [reply docs/index.mdx:2]: `Patrol – Flutter UI test framework`

A tiny bit shorter and more descriptive.

File: docs/getting-started/getting-started.mdx [hunk 5]
```diff
@@ -0,0 +1,23 @@
+---
+title: What you can use Patrol for?
+---
+
+# What you can use Patrol for?
+
+Patrol consists of 2 packages:
+
+- [patrol] – the core package
+- [patrol_cli] - the CLI app to run tests.
```
Comment (bartekpacia) [inline docs/getting-started/getting-started.mdx:10]: Currently it's:

```
Patrol consists of 2 packages:

[patrol] – the core package
[patrol_cli] - the CLI app which enables the [native automation feature]

...

[native automation feature]: https://patrol.leancode.co/native/overview
```

I think that `the CLI app to run tests` is not specific enough. Maybe:

```
the CLI program to run tests that use the [native automation feature]
```

or (but sounds a bit strange)

```
the CLI program to run tests using [native automation feature]
```

File: docs/getting-started/getting-started.mdx [hunk 6]
```diff
@@ -0,0 +1,23 @@
+---
+title: What you can use Patrol for?
```
Comment (bartekpacia) [inline docs/getting-started/getting-started.mdx:2]: I'd prefer to have this name (and the `#` heading below) match the name of the page in `docs.json`.

File: docs/getting-started/custom-finders.mdx [hunk 7]
```diff
@@ -1,13 +1,8 @@
 ---
-title: Getting started
+title: Getting started - custom finders
```
Comment (fylyppo) [inline docs/getting-started/custom-finders.mdx:2]: "Getting started" is redundant here. Maybe "Custom finders setup" or just "Custom finders"?

Comment (fylyppo) [reply docs/getting-started/custom-finders.mdx:2]: And title for `getting-started/native` is only "Setup". It could be named in the same convention.

**Comment by bartekpacia** [comment]
@jBorkowska I pushed commit [c6a6f8](https://github.com/leancodepl/patrol/pull/1239/commits/c6a6f80d4e7b82ff819a0ae3c61e65fc30c3c5b3) where I reorganized the Getting Started you wrote. Feel free modify it.

File: docs/getting-started/getting-started.mdx [hunk 8]
```diff
@@ -1,23 +1,37 @@
...
+[Custom finders setup] is much shorter than [native automation setup], since it
+only covers the setup needed to use our [custom finders feature] in your widget
+tests. Much of what it covers is already covered in [native setup].
 
 [patrol]: https://pub.dev/packages/patrol
 [patrol_cli]: https://pub.dev/packages/patrol_cli
-[Custom finders setup]: /getting-started/custom_finders
-[Native setup]: /getting-started/native
\ No newline at end of file
+[custom finders setup]: /getting-started/custom_finders
```
Comment (bartekpacia) [inline docs/getting-started/getting-started.mdx:34]: FYI these labels are case insensitive :)

File: docs/index.mdx (hunk 1 above)
Comment (jBorkowska) [reply docs/index.mdx:2]: We decided to leave `Main` for now

**Comment by bartekpacia** [comment]
LGTM. Thanks to all of you <3

**Pull request merged**